src/Consolonia.Core/Infrastructure/ConsoleWindow.cs

```diff
@@ -384,6 +394,9 @@ private void ConsoleOnMouseEvent(RawPointerEventType type, Point point, Vector?
                         (Vector)wheelDelta!, modifiers));
                     break;
             }
+
+            // draw the software cursor at this mouse position
+            RenderSoftwareCursor(point);
```

Comment:
Just out of curiosity: why are we thinking, that no render will happen after mousemove event? Becuase it's "input" priority in the dispatcher?